From: Bryan Donlan <bdonlan@gmail.com>
Date: Thu, 12 May 2011 18:58:32 -0400
Subject: [PATCH] Improve manpage whatis entries

This patch improves the whatis entries for the ofxdump and ofxconnect
manpages. First, CMDLINE_PARSER_PACKAGE is defined to the name of the
binary in question, thus replacing 'libofx' with eg 'ofxdump' in the
whatisline. Then, a -n argument is passed into help2man to define the
description portion of the manpages.
---
 ofxconnect/Makefile.am |    6 ++++--
 ofxdump/Makefile.am    |    5 +++--
 2 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/ofxconnect/Makefile.am b/ofxconnect/Makefile.am
index 4f1b402..72096d3 100644
--- a/ofxconnect/Makefile.am
+++ b/ofxconnect/Makefile.am
@@ -6,7 +6,9 @@ dist_man_MANS = ofxconnect.1
 AM_CPPFLAGS = \
 	-I${top_srcdir}/inc \
 	@LIBCURL_CPPFLAGS@ \
-	@LIBXMLPP_CFLAGS@
+	@LIBXMLPP_CFLAGS@ \
+	'-DCMDLINE_PARSER_PACKAGE="ofxconnect"'
+
 
 if USE_GENGETOPT 
 CLEANFILES = cmdline.c cmdline.h
@@ -25,4 +27,4 @@ TESTS = test-privateserver.sh
 
 ofxconnect.1: ofxconnect.cpp $(top_srcdir)/configure.in
 	$(MAKE) $(AM_MAKEFLAGS) ofxconnect$(EXEEXT)
-	help2man -N --output=ofxconnect.1 ./ofxconnect$(EXEEXT)
+	help2man -n 'Create a statement request file' -N --output=ofxconnect.1 ./ofxconnect$(EXEEXT)
diff --git a/ofxdump/Makefile.am b/ofxdump/Makefile.am
index a1d652f..811c580 100644
--- a/ofxdump/Makefile.am
+++ b/ofxdump/Makefile.am
@@ -4,7 +4,8 @@ ofxdump_SOURCES = cmdline.h cmdline.c ofxdump.cpp
 dist_man_MANS = ofxdump.1
 
 AM_CPPFLAGS = \
-	-I${top_builddir}/inc
+	-I${top_builddir}/inc \
+	'-DCMDLINE_PARSER_PACKAGE="ofxdump"'
 
 if USE_GENGETOPT 
 CLEANFILES = cmdline.c cmdline.h
@@ -16,7 +17,7 @@ endif
 
 ofxdump.1: ofxdump.cpp $(top_srcdir)/configure.in
 	$(MAKE) $(AM_MAKEFLAGS) ofxdump$(EXEEXT)
-	help2man -N --output=ofxdump.1 ./ofxdump$(EXEEXT)
+	help2man -n 'Dump content of OFX files as human-readable text' -N --output=ofxdump.1 ./ofxdump$(EXEEXT)
 
 MAINTAINERCLEANFILES  = cmdline.c cmdline.h
 EXTRA_DIST = cmdline.ggo
-- 
